src/schema/type/Annotation.graphql

```diff
@@ -0,0 +1,129 @@
+"An Annotation Target which refers to a node in the CE"
+type AnnotationCETarget {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID @id
+    "The item in the CE that this Annotation targets"
+    target: ThingInterface! @relation(name: "ANNOTATION_TARGET", direction: OUT)
+    "The name of the field of the item in the CE which contains the target value. If not set, the target refers to the entire node"
+    field: String
+    "In the case that the target is a fragment, the fragment value"
+    fragment: String
+
+    "http://purl.org/dc/terms/creator"
+    creator: String
+
+    "http://purl.org/dc/terms/created"
+    created: _Neo4jDateTime
+    "http://purl.org/dc/terms/modified"
+    modified: _Neo4jDateTime
+}
+
+"http://www.w3.org/ns/oa#TextualBody"
+type AnnotationTextualBody {
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID @id
+    "http://www.w3.org/1999/02/22-rdf-syntax-ns#value"
+    value: String!
+    "http://purl.org/dc/elements/1.1/format"
+    format: String
+    "http://purl.org/dc/elements/1.1/language"
+    language: AvailableLanguage
+    # Should always be "http://www.w3.org/ns/oa#TextualBody"
+    "http://www.w3.org/1999/02/22-rdf-syntax-ns#type"
+    type: String
+
+    "http://purl.org/dc/terms/creator"
+    creator: String
+
+    "http://purl.org/dc/terms/created"
+    created: _Neo4jDateTime
+    "http://purl.org/dc/terms/modified"
+    modified: _Neo4jDateTime
+}
+
+"http://www.w3.org/ns/oa#Annotation"
+type Annotation implements ThingInterface {
+    #############################
+    ### Annotation properties ###
+    "http://purl.org/dc/elements/1.1/identifier,https://schema.org/identifier"
+    identifier: ID @id
+    # An annotation can have multiple Targets, but must have at least one
+    "http://www.w3.org/ns/oa#hasTarget"
+    targetNode: [AnnotationCETarget] @relation(name: "ANNOTATION_TARGET", direction: OUT)
+    "An external resource that this annotation is about"
+    targetUrl:  [String]
+    # An annotation should always have a motivation. If you want to use a more specific
+    # annotation from a toolkit (a DefinedTerm) then you can include it, but you should
+    # still add the base motivation.
+    "http://www.w3.org/ns/oa#Motivation"
+    motivationDefinedTerm: DefinedTerm
```

Comment:
I think this is the only property left that is missing a relation, right?

Reply:
yes, added